homeassistant/components/rointe/sensor.py

```diff
@@ -0,0 +1,108 @@
+"""A sensor for the current Rointe radiator temperature."""
+from __future__ import annotations
+
+from datetime import datetime
+
+from rointesdk.device import RointeDevice
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ENERGY_KILO_WATT_HOUR, POWER_WATT, TEMP_CELSIUS
```

Comment:
Per the Actions warning, these are all deprecated in favor of `UnitOfEnergy`, `UnitOfPower`, and `UnitOfTemperature`, respectively.